src/index.html

```diff
@@ -77,24 +77,23 @@
         <p>
           DIM hasn't loaded. Something may be wrong with it, or with your browser (maybe you have a
           content blocker, or have disabled JavaScript, or your browser is too old). Try force
-          reloading to make sure (Ctrl-F5 or Cmd-R).
+          reloading to make sure (Ctrl-F5 or ⌘-Shift-R).
         </p>
         <ul>
-          <li><a href="https://twitter.com/ThisIsDIM">@ThisIsDIM on Twitter</a></li>
           <li>
             <a href="https://destinyitemmanager.fandom.com/wiki/Troubleshooting"
               >Troubleshooting Guide</a
             >
           </li>
+          <li><a href="https://twitter.com/ThisIsDIM">@ThisIsDIM on Twitter</a></li>
         </ul>
       </div>
     </div>
     <a
       href="https://github.com/DestinyItemManager/DIM/wiki/Supported-Browsers"
       id="browser-warning"
     >
-      Your browser is not supported by DIM. Some or all DIM functionality may not work. Please
-      switch to a supported browser.
+      The DIM team does not support using this browser. Some or all DIM functionality may not work.
```

Comment:
```suggestion
      The DIM team does not support using this browser. Some or all DIM features may not work.
```
grammatically speaking functionality is an uncountable noun/concept you can add or remove,
a "function" is a single thing another thing can perform.
but "function" and "work" being synonyms makes the sentence awkward so maybe this is best.